CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+> git remote add upstream https://github.com/Octasol/octasol.git
+> git fetch upstream
+> git branch --set-upstream-to=upstream/master master
+> ```
+>
+> This adds the original repository as a "remote" called "upstream," fetches the latest changes from that repo, 
+>and sets your local `master` to track the upstream `master` branch. To update your `master` branch, run `git pull`. 
+>Afterward, create new branches from `master` for your pull requests.
+
+<p>&nbsp;</p>
+
+## Pull Requests Process
+
+1. Fork the Repository
+1. Clone the Repository
+
+     ```
+    git clone https://github.com/<yourusername>/octasol.git <octasol>
+
+    cd <octasol>
+    ```
+
+1. Create a branch for your PR 
+
+    ```
+    git checkout -b <your-branch-name>
+    ```
+
+1. Commit, push changes to your PR 
+
+    ```
+    git add . 
+    git commit -m "Include a message about your PR"
+    git push origin <your-branch-name>
+    ```
+
+1. Open [Pull Request](https://docs.github.com/de/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)
+    * Go to your fork on GitHub, where you'll see a notification that your branch has recent pushes and an option to create a Pull Request. 
+    * Click on the Compare & Pull Request button and select your branch from the "head repository" dropdown, and compare it with the main or master branch of the original repository.  
+    * Provide a title and clear description for your pull request and which issue it fixes. 
+    * Prefer async/await over promises for handling asynchronous operations, as it improves readability.
+
+1. Pull Request Review  
+    * One or more maintainers will review the changes and may request modifications. After approval, your PR will be merged into the main codebase.
+
+> [!NOTE] 
+> Respond to feedback: If the maintainer requests changes, commit those changes to the same branch. Push the updated code, and GitHub will automatically update your PR.
+> Approval: Once the changes are satisfactory, the reviewer will approve the PR.
+
+
+<p>&nbsp;</p>
+
+## Code Guidelines
+
+* Use Consistent Naming Conventions
+* Use camelCase for variables and function names (e.g., myVariable, calculateTotal)
+* Use PascalCase for class names and components (e.g., MyComponent, UserProfile).
+* Use UPPER_SNAKE_CASE for constants (e.g., MAX_USERS).
+* Write clear and concise comments to explain complex logic.
+* Leverage TypeScript's features like interfaces, enums, and type annotations to enhance type safety.
+* Use strict mode for stricter type checking ("strict": true in tsconfig.json).
+* Use OOP concepts for code organization and reuse.
+* Use .env.local for environment variables to separate secrets and configurations from your codebase.
+* Use Tailwind’s responsive utilities to build responsive designs easily.
+* Customize your theme in the tailwind.config.js file to define your own color palette, spacing, and more.
+
+<p>&nbsp;</p>
+
+## Test your Code
```

Comment:
We haven't added tests for now, so this can be removed

Reply:
Thanks for the feedback. I pushed a new commit considering your feedback